services/motion/client_test.go

```diff
@@ -228,4 +235,335 @@ func TestClient(t *testing.T) {
 		test.That(t, client2.Close(context.Background()), test.ShouldBeNil)
 		test.That(t, conn.Close(), test.ShouldBeNil)
 	})
+
+	t.Run("MoveOnGlobeNew", func(t *testing.T) {
```

Comment:
I added test cases for both success & error cases for all the new methods. Figure its important as the client code does do some massaging of inputs & outputs.